app/controllers/katello/api/v2/content_view_components_controller.rb

```diff
@@ -114,5 +120,19 @@ def component_params
       attrs = [:latest, :content_view_version_id, :content_view_id]
       params.require(:content_view_component).permit(*attrs)
     end
+
+    def default_sort
+      %w(label asc)
+    end
+
+    def sort_options
+      if default_sort.is_a?(Array)
+        return [default_sort[0], default_sort[1], {}]
+      elsif default_sort.is_a?(Proc)
+        return [nil, nil, { :custom_sort => default_sort }]
+      else
```

Comment:
Not a big deal, but by the time an `elsif` is needed I kinda prefer `case`:
```rb
case default_sort
  when Array
    ...
  when Proc
   ...
  else
  ...
  end
```
Since Ruby cases don't have fall-through, you also wouldn't have to say `return` or `break` each time.

Reply:
Updated..